app/views/layouts/application.html.erb

```diff
@@ -17,7 +17,7 @@
     :rss, observer_rss_path, { title: :app_rss.l }
   ) %>
   <title><%= :app_title.l%>: <%= title_tag_contents(controller.action_name)%></title>
-  <link rel="SHORTCUT ICON" href="https://mushroomobserver.org/favicon.ico"/>
+  <link rel="SHORTCUT ICON" href="/favicon.ico?20220116"/>
```

Comment:
* I removed the protocol and domain in order to make this work locally. I think it will also work on the webserver. 
**Is there a reason why it won't work on the webserver?**

* I used an ico instead of the png that Jeff sent me. (I converted Jeff's file to an ico.):
  * I couldn't figure out how to make the png work. (I tried to make appropriate changes from the png example in https://en.wikipedia.org/wiki/Favicon#How_to_use, but couldn't get any browser to show the favicon.)
  * Maybe that's for the best: IE < v11 won't work with a png. 
* The query forces the browser to refresh the favicon. See https://stackoverflow.com/questions/2208933/how-do-i-force-a-favicon-refresh